go.mod

```diff
@@ -0,0 +1,18 @@
+module gitlab.alibaba-inc.com/cloudnativeapp/oam-runtime
```

Comment:
is this supposed to be an alibaba module?

Reply:
Good catch. Fixing it shortly.